### PR TITLE
Split the long Content.ModeKind texts into lines

### PR DESCRIPTION
**Pull request by kayvank**
This pull request is a fix for issue-283.

### CODE REVIEW

File: CHANGELOG.md
Comment (Mikolaj) [inline CHANGELOG.md:3]: Thank you for thinking about the changelog. LambdaHack v0.11.0.0 has already been released. Please start a new section (v0.11.1.0) and move your item there.

File: GameDefinition/Content/ModeKind.hs
Comment (Mikolaj) [inline GameDefinition/Content/ModeKind.hs:129]: A space between `[` and `"Run` would give it a more uniform style, given the space between comma and text below. And similarly elsewhere.

File: GameDefinition/Content/ModeKind.hs
Comment (Mikolaj) [inline GameDefinition/Content/ModeKind.hs:286]: I think the formatting with `where` on a separate one that you used before, is better.

**Review by Mikolaj — APPROVED** [review]
Well done.

**Comment by kayvank** [comment]
Thank you @Mikolaj for reviewing,  approving, and helping me with this PR. 
I've implemented your suggestions. Is there anything else I need to do to get the "workflow approval" required for the merge?

**Comment by Mikolaj** [comment]
No, the "workflow approval" is because you are first-time commiter. Your next PR should run CI automatically, without any approval needed. As soon as CI finishes, I will squash and merge. BTW, you could have amended the existing commit (`git commit --amend -a`) and done `push -f`. But no problem, squash and merge will make these commits into one.

**Pull request merged**
